app/static/styling-css/common-elements-style.css

```diff
@@ -1,4 +1,3 @@
-/* Defining css custom colour properties (variables) within root selector*/
```

Comment:
Many lines of code are deleted from this file. If you are not adding anything in this file. No need to push it. Also, try taking the latest pull from main branch before committing if not taken already.